products/idn/tools/cli/index.md

```diff
@@ -0,0 +1,210 @@
+---
+id: cli
+title: CLI
+pagination_label: CLI
+sidebar_label: CLI
+sidebar_position: 1
+sidebar_class_name: cli
+keywords: ['cli']
+description: Learn how to use the SailPoint CLI in this guide. 
+slug: /tools/cli
+tags: ['CLI']
+---
+
+## Start using the CLI
```

Comment:
We should probably have a section before this that explains what the CLI is, for the uninitiated.